claragenomics/dl4atac/utils.py

```diff
@@ -51,6 +51,25 @@ def make_experiment_dir(label, out_home, timestamp=True, create_latest_symlink=T
         os.symlink(os.path.abspath(exp_path), latest_symlink)
     return exp_path
 
+def save_config(exp_dir, root_dir, config_file_path=None):
+    """
+    Save config files used for training. 
+    Args:
+        exp_dir : Directory where the models are being saved.
+        root_dir : Directory where the main script resides. It's assumed that the configs
+                   directory is at the same level as the script.
+        config_file_path : Path to a custom config file if any.
+    """
+    myprint("Saving config files to {}...".format(exp_dir))
+    # Save custom config file if provided
+    if config_file_path:
+        print ("comes here")
+        shutil.copy(config_file_path, exp_dir)
+    # Save default config
+    config_path = os.path.join(root_dir, "configs")
+    dst_path = os.path.join(exp_dir, "configs")
+    shutil.copytree(config_path, dst_path)
```

Comment:
will this copy the whole config folder? Would be better to copy only those config files used for the run

Reply:
copying just the model_structures.yaml for now. Will make it automatic in the next PR.